<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>withdraw</class>
 <widget class="QDialog" name="withdraw">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn100e">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>300</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>100e</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn50e">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>340</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>50e</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn20e">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>390</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>20e</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn10e">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>430</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>10e</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>270</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>valitse summa:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineSumma">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>280</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>130</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tilit√§si:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTeeNosto">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>340</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Tee nosto</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSaldo">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>160</y>
     <width>441</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>saldo:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCredit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>190</y>
     <width>451</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>credit:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLogOut">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>440</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Kirjaudu ulos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Takaisin</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNomoney">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>340</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
